themes/default/content/blog/deploy-to-multiple-regions/index.md

```diff
@@ -0,0 +1,759 @@
+---
+title: "Deploy Infrastructure to Multiple Cloud Regions at Once"
+date: 2022-08-03T08:00:00-07:00
+meta_desc: "Use infrastructure as code to deploy to multiple regions, accounts, or clusters at the same time, using Pulumi explicit provider configuration."
+meta_image: deployinfra.png
+authors: ["joe-duffy"]
+tags: ["aws", "regions", "rds"]
+---
+
+Pulumi makes it easy to flexibly deploy your cloud infrastructure using code. Usually deployments encomass a single slack and a single region in your cloud of choice. If you need to go multi-region, that usually simply means creating a stack per-region, which Pulumi's configuration system makes easy. A stack per region isn't required, though! Sometimes we want a single stack to span regions for performance, scalability, resilience, or just hard requirements. In these cases, Pulumi can seamlessly orchestrate deployments to, or even across, multiple regions,  accounts, or clusters. In this article, we'll see this in action by provisioning an AWS RDS primary database into one region and a read replica in an entirely different region -- all from a single Pulumi program, stack, and `pulumi up` incantation.
```

Comment:
```suggestion
Pulumi makes it easy to flexibly deploy your cloud infrastructure using code. Usually deployments encompass a single slack and a single region in your cloud of choice. If you need to go multi-region, that usually means creating a stack per-region, which Pulumi's configuration system makes easy. A stack per region isn't required, though! Sometimes we want a single stack to span regions for performance, scalability, resilience, or just hard requirements. In these cases, Pulumi can seamlessly orchestrate deployments to, or even across, multiple regions,  accounts, or clusters. In this article, we'll see this in action by provisioning an AWS RDS primary database into one region and a read replica in an entirely different region -- all from a single Pulumi program, stack, and `pulumi up` incantation.
```

we added avoiding `simply` and `easy` to our style guide a bit ago, i think ur use of easy is tots okay here though, would suggest dropping simply though.